e2e/.eslintrc

```diff
@@ -11,5 +11,13 @@
     "cypress/globals": true,
     "node": true
   },
-  "plugins": ["cypress"]
+  "plugins": ["cypress"],
+  "overrides": [
+    {
+      "files": ["*.cy.spec.*"],
+      "rules": {
+        "no-console": "error"
```

Comment:
This used to be disabled. I suspect the reason is we have some legit uses of `console.log` inside `e2e` folder, but in the actual tests file, we should never allow that.